src/utils/constants.ts

```diff
@@ -0,0 +1,10 @@
+export const CONSTANTS = {
+  SOOS: {
+    ENVIRONMENT_VARIABLES: {
```

Comment:
we should switch to the casing on the api client for consistency, e.g. 
SOOS: {
  EnvironmentVariables: {
    ApiKey: "SOOS_API_KEY"